Checkmate2019/Base/models.py

```diff
@@ -24,7 +23,3 @@ class Member(models.Model):
     
     def __str__(self):
         return f"{self.team} : {self.id}"
```

Comment:
Return team's name, not the team object

Reply:
That would not make any difference. The string representations for team object is team name. So even if for the member object I return team name and not team object as string representation, it would not make any difference.

Reply:
Definitely, it doesn't make any difference in execution. But it only appears more logical and less misleading. Also, you might run into problems somewhere else down the line. 
Just a friendly advice :smiley: